Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
I personally prefer to use lightweight docker image like anapsix/alpine-java:8, but for better security and support, is it possible to use instead the official java image java:8-jre ?